public/js/apps/management/main.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var $ = require('jquery');
+var App = require('./app');
+
+// Common ajax settings.
+$.ajaxSetup({
+  dataType: 'json',
```

Comment:
hmm, it's possible that the [header config](https://github.com/mozilla/payments-ui/blob/master/public/js/actions/user.jsx#L25) might erase this setting.